employee_project/employee.py

```diff
@@ -86,3 +86,24 @@ def get_emp_salary_input(msg = "Enter employee salary:"):
         print("Invalid input, enter a valid input ex: 20000")
         sal = get_emp_salary_input(msg)
     return sal
+
+
+def employee_name(employee_name):
+    """ @param: string
+        @desc: method returns the valide name
```

Comment:
change the method description:  This Method receives the input from the user and validating the name .

Reply:
changed the method description